src/components/Login.jsx

```diff
@@ -0,0 +1,58 @@
+import { GoogleAuthProvider, signInWithPopup } from "firebase/auth";
+import { useQuery, useMutation } from "@tanstack/react-query";
+import { useNavigate } from "react-router-dom";
+import { firebaseAuth } from "./app/firebaseAuth";
+import logo from "../assets/dataface_logo.png";
+import googleLogo from "../assets/google_logo.png";
+import fetchData from "../utils/axios";
+
+function Login() {
+  const navigate = useNavigate();
+  const googleProvider = new GoogleAuthProvider();
+
+  async function handleGoogleLogin() {
+    const result = await signInWithPopup(firebaseAuth, googleProvider);
+    const { user } = result;
```

Comment:
```suggestion
    const user = await signInWithPopup(firebaseAuth, googleProvider);
```
혹시 이부분 구조분해 할당 없이 바로 할당하는건 어려울까요? 
가능하다면 줄여보는것도 괜찮지 않을까 생각해봅니다! 👍

Reply:
반영하였습니다!